src/components/Pages/OfficialDocuments/OfficialDocumentCollection.js

```diff
@@ -4,7 +4,7 @@ import { injectIntl } from 'react-intl';
 
 import ContextualNav from 'components/PageSections/ContextualNav';
 import PageHeader from 'components/PageHeader';
-import OfficialDocumentPage from 'components/Pages/OfficialDocuments/OfficialDocumentPage';
+import OfficialDocumentPaginationPage from 'components/Pages/OfficialDocuments/OfficialDocumentPaginationPage';
```

Comment:
Renamed it, I'm open to other suggestions though!

Reply:
I like this name.